language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/diagnostic/DiagnosticsHelper.java

```diff
@@ -151,9 +152,9 @@ public synchronized void compileAndSendDiagnostics(ExtendedLanguageClient client
      * @param compilation package compilation
      */
     private synchronized void compileAndSendDiagnostics(ExtendedLanguageClient client, Path projectRoot,
-                                                        PackageCompilation compilation) {
+                                                        PackageCompilation compilation, WorkspaceManager workspace) {
```

Comment:
Better if we can change `workspace` to `workspaceManager`

Reply:
Done